<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GGSearchDialog</class>
 <widget class="QDialog" name="GGSearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Find:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cmbSearch">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="maxCount">
      <number>10</number>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAtTop</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Types</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="chkEverything">
        <property name="text">
         <string>Everything</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="chkPageCaption">
        <property name="text">
         <string>Page Caption</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="chkFunctions">
        <property name="text">
         <string>Functions</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="chkVariables">
        <property name="text">
         <string>Variables</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="chkLinkName">
        <property name="text">
         <string>Links</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QCheckBox" name="chkPageScene">
        <property name="text">
         <string>Page Scene</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="chkPageName">
        <property name="text">
         <string>Page Name</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="chkContent">
        <property name="text">
         <string>Content</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QRadioButton" name="radContains">
        <property name="text">
         <string>Contains</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QRadioButton" name="radWholeWord">
        <property name="text">
         <string>Whole Word</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="radStartsWith">
        <property name="text">
         <string>Starts With</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="chkCaseSensitive">
        <property name="text">
         <string>Case Sensitive</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QRadioButton" name="radExact">
        <property name="text">
         <string>Exact</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>New Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GGSearchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GGSearchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkEverything</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkPageCaption</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkEverything</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkPageName</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkEverything</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkPageScene</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkEverything</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkLinkName</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkEverything</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkContent</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkEverything</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkVariables</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkEverything</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkFunctions</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>GGSearchDialog</receiver>
   <slot>resetSearch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>resetSearch()</slot>
 </slots>
</ui>
